Assets/Plugins/EuclideonUdSDK/udSDK/UDCameraOptions.cs

```diff
@@ -86,19 +85,19 @@ void Update()
                     {
                         var marker = GameObject.CreatePrimitive(PrimitiveType.Cube);
                         marker.GetComponent<Renderer>().material.color = Color.red;
-                        marker.transform.position = pickCentre;
+                        marker.transform.position = pick.pointCenter;
                     }
                 }
-                previewCube.transform.position = pickCentre;
+                previewCube.transform.position = pick.pointCenter;
             }
             placeNext = false;
         }
 
         Vector3 mp = Input.mousePosition;
-        optionsStruct.setPick((uint)(mp.x * resolutionScaling), (uint)((cam.pixelHeight - mp.y)*resolutionScaling));
```

Comment:
This was intentionally set outside the click event so that the user could query the pick position at any time, not just after a click